### PR TITLE
Creating images directly from createImage API

### PR DESCRIPTION
**Pull request by abchatra**
reduces memory pressure

### CODE REVIEW

File: libs/core/icons.ts [hunk 1]
```diff
@@ -194,317 +194,317 @@ namespace basic {
 
 namespace images {
 
-    function getArrow(i : ArrowNames): string {
+    function getArrow(i : ArrowNames): Image {
```
Comment (pelikhan) [inline libs/core/icons.ts]: do we even need "getArrow"?

Comment (abchatra) [inline libs/core/icons.ts]: image

Comment (pelikhan) [inline libs/core/icons.ts]: I mean, what about moving the content of getArrow under showArrow.

**Comment by mmoskal** [comment]
Lgtm

**Pull request merged**